worker/lib/workers/testbot.ts

```diff
@@ -11,19 +16,28 @@ import NetworkManager, { Supported } from '../helpers/nm';
 
 const dutSerialPath = '/reports/dut-serial.txt';
 
+const resolveDeviceInteractor = (hat: TestBotHat): DeviceInteractor => {
+	if (process.env.WORKER_DUT_TYPE === 'intel-nuc') {
```

Comment:
@roman-mazur this should be `E2E_DUT_TYPE` no?

Reply:
E2E_DUT_TYPE is used by the e2e test app, here we have a separate app, so I went with a separate variable.
But  I think it's a good idea to unify them since we move devices between apps quite often.

Reply:
@roman-mazur Yes, that way folks can set it once and ideally forget about their DUT type associated with each testbot (That's us). Later for folks having their own standalone testbots, I think we can specify this as a fleet-wide variable in applications and mention it in our getting started guide for it to be changed when they provision testbots for the first time.

Reply:
What is a good name then? TESTBOT_DUT_TYPE?

Reply:
Yes, `TESTBOT_DUT_TYPE` seems like a good fit for both e2e and leviathan as well as the code in the SDK. One variable to be set to specify your DUT type.

Reply:
Updated.

Reply:
Resolving